### PR TITLE
Update release body with generated changelog

### PR DESCRIPTION
**Pull request by HotThoughts**
- Add generate changelog action
- Update release body with generated changelog

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -27,6 +29,30 @@ jobs:
...
+          "https://api.github.com/repos/${{ github.repository }}/releases/latest" | jq '.id')
+
+        RELEASE_BODY="${{ steps.changelog.outputs.changelog }}"
+        RELEASE_BODY="${RELEASE_BODY//$'\n'/'\n'}"
+
+        # Update release body
+        curl -X PATCH -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+          -H "Accept: application/vnd.github.v3+json" \
+          "https://api.github.com/repos/${{ github.repository }}/releases/${RELEASE_ID}" \
+          -d "{\"body\":\"$RELEASE_BODY\"}"
```
Comment (tczekajlo) [inline .github/workflows/release.yml:58]: It should be a separate workflow that runs only for a push tag event.

File: README.md [hunk 2]
```diff
@@ -97,6 +97,32 @@ are merged. The only required steps are:
 2. Run `helm lint --strict charts/rasa-x`
 3. Increase the chart `version` in `charts/rasa-x/Chart.yaml`
 
+### Constructing Change logs
```
Comment (tczekajlo) [inline README.md:100]:
```suggestion
### Changelog
```

**Review by tczekajlo — CHANGES_REQUESTED** [review]

File: .github/workflows/changelog.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# - Update the latest release body with generated changelogs
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
```
Comment (tczekajlo) [inline .github/workflows/changelog.yml:13]: Almost there, you miss the `checkout` step.

```
    - name: Checkout
      uses: actions/checkout@v2
      with:
        fetch-depth: 0
```

**Review by tczekajlo — CHANGES_REQUESTED** [review]

File: .github/workflows/changelog.yml (hunk 3 above)
Comment (HotThoughts) [reply .github/workflows/changelog.yml:13]: Thank you! I added checkout in `changelog.yml`. We need the full history to generate changelog but not for release, right?

**Review by tczekajlo — APPROVED** [review]
🎖️

**Pull request merged**